libtool/PKGBUILD

```diff
@@ -1,105 +1,106 @@
+            'a7193916ecd935347774b9e1059fea0318bc6ca1d5500fcc1095cf9d23c04dc9'
+            '637b5a6bc43b725ee0ad789941ff5d655ade888bee0de2c31fecdb890fd023d7'
+            'a0c39cbf1658f3619212ccc403b190f347303f5559a9fa47642347f29230b30d'
+            'af2ef1351005ccc0ddc6b1773fb6bd502827d91efe7e3acf2e87dcabbb0e6a25'
             'c95a65e890b1ae6362807abc66809e72cf81aeea5f9f556e38f9752f974bf435'
             '8069e887aeeab7491f15e00547fa66d9b9e86407f5a23f37a6d8c7d165de752e'
-            'db16cd322e0ebc578c906e94b0788810af17ce617c700a50db2e3c598dbbed7e'
-            '9fe454603a519e9ac50b32de125d4c5293520142c34a895c6e392a190719ff52')
-
- validpgpkeys=('CFE2BE707B538E8B26757D84151308092983D606'  # Gary Vaughan (Free Software Developer) <gary@vaughan.pe>
-               '7C5FBB96BE82B954AC20DF5F6EAC957F8EEB55C0') # Alex Ameen <alex.ameen.tx@gmail.com>
+            'db16cd322e0ebc578c906e94b0788810af17ce617c700a50db2e3c598dbbed7e')
+validpgpkeys=('7C5FBB96BE82B954AC20DF5F6EAC957F8EEB55C0'  # Alex Ameen <alex.ameen.tx@gmail.com>
+              'FA26CA784BE188927F22B99F6570EA01146F7354') # Ileana Dumitrescu <ileanadumi95@protonmail.com>
 
 # Helper macros to help make tasks easier #
 apply_patch_with_msg() {
-  for _patch in "$@"
-  do
+  for _patch in "$@"; do
     msg2 "Applying $_patch"
-    patch -Nbp1 -i "${srcdir}/$_patch"
+    patch -Nbp1 -i "${srcdir}/${_patch}"
   done
 }
+# =========================================== #
 
 prepare() {
-  cd ${srcdir}/${pkgbase}-${pkgver}
+  cd "${pkgbase}-${pkgver}"
 
+  # Note: keep the patches in sync with mingw libtool
   apply_patch_with_msg \
-    0002-cygwin-mingw-Create-UAC-manifest-files.mingw.patch \
-    0003-Pass-various-runtime-library-flags-to-GCC.mingw.patch \
-    0005-Fix-seems-to-be-moved.patch \
-    0006-Fix-strict-ansi-vs-posix.patch \
-    0007-fix-cr-for-awk-in-configure.all.patch \
-    \
-    0009-libtool-2.4.2.418-msysize.patch \
-    0010-libtool-2.4.2-include-process-h.patch \
-    0011-Pick-up-clang_rt-static-archives-compiler-internal-l.patch \
-    0012-Prefer-response-files-over-linker-scripts-for-mingw-.patch \
-    0013-Allow-statically-linking-compiler-support-libraries-.patch \
-    0014-Support-llvm-objdump-f-output.patch \
-    0015-Allow-statically-linking-Flang-support-libraries.patch
+    0001-cygwin-mingw-Create-UAC-manifest-files.patch \
+    0002-Fix-seems-to-be-moved.patch \
+    0012-Prefer-response-files-over-linker-scripts-for-mingw.patch \
+    0013-Allow-statically-linking-compiler-support-libraries.patch \
+    0014-Support-llvm-objdump-f-output.patch
 
+  # MSYS2-specific patches
+  apply_patch_with_msg \
+    0005-libtool-include-process.h.patch \
+    0006-msysize.patch
+
+  autoconf -f
 }
 
 build() {
-  cd ${srcdir}/${pkgbase}-${pkgver}
-  ./configure --prefix=/usr lt_cv_sys_lib_dlsearch_path_spec="/usr/lib /lib"
-  make -j1
+  cd "${pkgbase}-${pkgver}"
+
+  ./configure \
+    --prefix=${MINGW_PREFIX} \
```

Comment:
why `${MINGW_PREFIX}` is used here?

Reply:
Oops, it's my mistake.

Reply:
I'll fix it in the end of the conversation.